Sources/Profile-View.php

```diff
@@ -393,7 +393,7 @@ function fetch_alerts($memID, $all = false, $counter = 0, $pagination = array(),
 			{
 				$alerts[$id_alert]['extra']['msg_msg'] = $txt['topic_na'];
 				$alerts[$id_alert]['extra']['topic_msg'] = $topic_msgs[$alert['extra']['topic']][$alert['show_links'] ? 'link' : 'text'];
-				$alerts[$id_alert]['extra']['board_msg'] = isset($alert['extra']['board']) ? $board_msgs[$alert['extra']['board']][$alert['show_links'] ? 'link' : 'text'] : $txt['board_na'];
+				$alerts[$id_alert]['extra']['board_msg'] = isset($alert['extra']['board']) && isset($board_msgs[$alert['extra']['board']]) ? $board_msgs[$alert['extra']['board']][$alert['show_links'] ? 'link' : 'text'] : $txt['board_na'];
```

Comment:
This will cause undefined errors if for some reason $board_msgs[$alert['extra']['board']] ends up being an empty array, it will pass the isset check but will show an undefined error because it will not have any key on it.

Ternaries are useful but it can quickly become a mess if more checks are added, therefor it might be better to declare a var and perform all the needed checks on it, once we've done that just pass the var to $alerts[$id_alert]['extra']['board_msg']